src/libraries/Microsoft.Bcl.Numerics/src/Resources/strings.resx

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please rename this file to `Strings.resx` (capital S): https://github.com/dotnet/runtime/blob/ea2022818b117bead26fc19c8ebb47b455907cab/eng/resources.targets#L3